extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -11,9 +11,34 @@ class ImageReviewHelper extends ImageReviewHelperBase {
 
 	private $user_id = 0;
 
+	private $image_stats_cache_keys = [];
+
+	const STATS_REJECTED = 'rejected';
+
+	const STATS_UNREVIEWED = 'unreviewed';
+
+	const STATS_QUESTIONABLE = 'questionable';
+
+	const STATS_INVALID = 'invalid';
+
+	const STATS_REVIEWER = 'reviewer';
+
 	function __construct() {
 		parent::__construct();
 		$this->user_id = $this->wg->user->getId();
+
+		$stats_keys = [
+			self::STATS_REJECTED, self::STATS_UNREVIEWED, self::STATS_QUESTIONABLE,
+			self::STATS_INVALID, self::STATS_REVIEWER
+		];
+		foreach ( $stats_keys as $key ) {
```

Comment:
This loop has logic that for one element from list is different.
I suggest separate this loop for all keys except `STATS_REVIEWER` and add `STATS_REVIEWER` outside loop.